website/www/site/content/en/blog/gsoc-25-yaml-user-accessibility.md

```diff
@@ -0,0 +1,110 @@
+---
+title:  "Google Summer of Code 2025 - Beam YAML, Kafka and Iceberg User
+Accessibility"
+date:   2025-09-23 00:00:00 -0400
+categories:
+  - blog
+  - gsoc
+aliases:
+  - /blog/2025/09/23/gsoc-25-yaml-user-accessibility.html
+authors:
+  - charlespnh
+
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+The relatively new Beam YAML SDK was introduced in the spirit of making data processing easy,
+but it has gained little adoption for complex ML tasks and hasn’t been widely used with Managed IOs such as Kafka and Iceberg.
+As part of Google Summer of Code 2025, new illustrative, production-ready pipeline examples of ML use cases with Kafka and Iceberg
```

Comment:
Managed I/O (here and elsewhere)

Also let's add a link to https://beam.apache.org/documentation/io/managed-io/.